transactions/transaction_services.py

```diff
@@ -14,7 +14,7 @@ class TransactionServices:
     """A helper class for online payment services"""
 
     rave_public_key = os.getenv('RAVE_PUBLIC_KEY')
-    rave_secret_key = os.getenv('RAVE_SECRET_KEY')
+    rave_secret_key = os.getenv('RAVE_TEST_SECRET_KEY')
```

Comment:
@shemaeric The right name of the environment variable is `RAVE_SECRET_KEY`. Do not override it.